x/ecocredit/basket/features/msg_create.feature

```diff
@@ -132,7 +132,7 @@ Feature: MsgCreate
     }
     """
     When the message is validated
-    Then expect the error "name must start with an alphabetic character, and be between 3 and 8 alphanumeric characters long: invalid request"
+    Then expect the error "name: expected format [a-zA-Z][a-zA-Z0-9]{2,7}: parse error: invalid request"
```

Comment:
perhaps we should opt for a more human readable error for users who might not understand regex?

Reply:
Yea, regex is probably not the most user friendly. :+1:

Reply:
https://github.com/regen-network/regen-ledger/pull/1392/commits/6d67b15b6e384a28d25ef959607733704b04d1be